packages/components/src/handler.ts

```diff
@@ -49,6 +58,7 @@ export class ConsoleCallbackHandler extends BaseTracer {
     constructor(logger: Logger) {
         super()
         this.logger = logger
+        logger.level = getEnvironmentVariable('LOG_LEVEL') ?? 'info'
```

Comment:
we might not need to specify this, as its already specified [here](https://github.com/FlowiseAI/Flowise/blob/bugfix/Remove-Doc-Store-File/packages/server/src/utils/config.ts#L12)

Reply:
When I tried to run with log level verbose, handler.ts used value info instead of verbose. Log level verbose just worked when I put this line.

Reply:
@HenryHengZJ I just check again and when I remove line 61, application stop to log verbose (or others levels), using default value info to log.

Reply:
if you try to set these env variables:
```
DEBUG=true
LOG_LEVEL=debug
```
you should be able to see the verbose logs no?

Reply:
I did it, but logs were shown multiline. DEBUG=true enables Langchain logs. It would be very good have these logs multiline to do a best ingestion in log analysis tools.